CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 project(VulkanBootstrap)
 
+option(VK_BOOTSTRAP_DISABLE_WARNINGS "Enable warnings as errors during compilation" OFF)
```

Comment:
Possibly copy/paste mistake? (the description is the same as the other option)

Reply:
Sure! Sorry for the copypaste haha. Updated!